woof-code/rootfs-petbuilds/connman-gtk/petbuild

```diff
@@ -10,4 +10,5 @@ build() {
     meson --buildtype=release --prefix=/usr -Duse_status_icon=false build
     ninja -C build install
     rm -rf /usr/share/applications
-}
\ No newline at end of file
+    cp -r ../usr/share/doc/* /usr/share/doc/
```

Comment:
You can remove this change, I'll fix this in petbuilds.sh. I'll make it add the extra files after documentation is moved out of the package.

Reply:
Doing this.